learning_resources/serializers.py

```diff
@@ -103,29 +147,233 @@ class LearningResourceBaseSerializer(serializers.ModelSerializer):
     audience = serializers.ReadOnlyField()
     certification = serializers.ReadOnlyField()
     prices = serializers.ReadOnlyField()
+    topics = WriteableSerializerMethodField()
+    course = CourseSerializer(read_only=True, allow_null=True)
+    learning_path = LearningPathSerializer(read_only=True, allow_null=True)
+    runs = LearningResourceRunSerializer(read_only=True, many=True, allow_null=True)
+    learningpath_items = serializers.SerializerMethodField()
```

Comment:
How about just `learning_paths`?

Reply:
I was thinking that `learning_path` (for the "extra details") and `learning_paths` (for the parents) might be confusing. But I'm good with it if you like that better. *It's not actually confusing on our frontend, imo, since the typehints will be very different.*